docs/manual/mainloop.rst

```diff
@@ -111,7 +111,15 @@ This event loop integrates with the asyncio module in Python.
     Use instead method :method:`run_in_executor` of :class:`AsyncioEventLoop`,
     which forward API of the same method from asyncio Event Loop run_in_executor_.
 
-.. _run_in_executor: https://docs.python.org/3.11/library/asyncio-eventloop.html#asyncio.loop.run_in_executor
+.. warning::
```

Comment:
make warning, because we do not want to read bugs "I used incompatible event loop and it crashed with AttributeError"